composer/workflows/dataflowtemplateoperator_create_dataset_and_table_test.py

```diff
@@ -0,0 +1,63 @@
+from google.cloud import bigquery
+import pytest
+import google.cloud.exceptions
+import uuid
```

Comment:
I thought there was a rule on import order. For this case, I would do:

```python
import uuid

from google.cloud import bigquery
import google.cloud.exceptions
import pytest
```

Reply:
Sorry, I still see mixed import statements.

Let me explain. It is a common practice to have 3 blocks of import statements.
1) std libraries
2) 3rd party libraries
3) local imports

Also, pytest and google.cloud.exceptoions are not used,
so for this case, I would do:

```python
import os
import uuid

from google.cloud import bigquery

from . import dataflowtemplateoperator_create_dataset_and_table_helper as helper
```

Reply:
I fixed this